src/courses/mappers/05.md

```diff
@@ -0,0 +1,53 @@
+---
+order: 5
+next: 06.md
+title: OHDF Schema Basics
+author: Charles Hu
+---
+
+## An Overview of the OHDF Schema
+
+In order to make an OHDF mapper, it is important to understand the specifics of how OHDF is actually structured through the OHDF schema.
+
+The OHDF schema is designed to provide a simple, structured, and hierarchal view of security tool results. OHDF actually consists of several related schema formats; however, the <i>Exec JSON</i> format is the most common and is typically the one being referred to when speaking of the OHDF schema. Any file or object that implements the schema can be broken down into a hierarchy of four main structures. These structures are:
+
+### 1. <i>**Exec JSON**</i>
+
+This is the top level structure that contains both the security data (which is stored in the <i>Profiles</i> structure) and relevant metadata on the tool used to generate the OHDF file (stored in structures such as <i>Platform</i>, <i>Statistics</i>, and <i>Version</i>). 
+
+### 2. <i>**Profiles**</i>
+
+This structure contains metadata on the target system of the original security tool export and on the specific run performed by the security tool. <i>Profiles</i> provide a high-level overview of the security tool result and the targeted system, which are formatted in a manner which is digestible and immediately accessible to the assessor. There is typically only one <i>profiles</i> structure. Further instances of <i>profiles</i> are additional overlays.
+
+### 3. <i>**Controls**</i>
+
+Controls are security requirements that are used to prevent, mitigate, and address various security risks to sensitive information and infrastructure. In the case of OHDF, the <i>controls</i> structure is a collection of requirements tested for by an external security tool to ensure that the target complies with vulnerability and weakness prevention standards. Any given <i>profile</i> contains some number of <i>controls</i> which were tested against the target system during the original security tool run.
+
+### 4. <i>**Results**</i>
+
+The <i>results</i> structure contains information on the results of specific tests ran by the security tool on the target system against a security requirement/control. These results will always correlate to a certain <i>control</i> and will either report `passed` or `failed` to indicate the test status (other statuses include `skipped` and `error`) which cumulatively influence the determined compliance level of the target system for some set of requirements/controls. Any given <i>control</i> contains some number of <i>results</i> which reflect the implemented tests to check if the target system is actually compliant.
```

Comment:
Change "...is actually compliant."
to: "...is actually meeting the overall requirement/control."

Reply:
Addressed.